eodag/resources/providers.yml

```diff
@@ -2121,6 +2124,9 @@
       class:
         - class
         - '$.class'
+      date:
+        - date
+        - '$.date'
```

Comment:
I am not sure the `date` field is needed : dates are already usable through `start` and `end` parameters using these settings:
```yml
startTimeFromAscendingNode: '{$.startTimeFromAscendingNode#to_iso_date}'
completionTimeFromAscendingNode:
  - 'date={startTimeFromAscendingNode#to_iso_date}/{completionTimeFromAscendingNode#to_iso_date(-1,)}'
  - '{$.completionTimeFromAscendingNode#to_iso_date}'
```